cuslide2_build_env.yaml

```diff
@@ -0,0 +1,56 @@
+name: cuslide2-build
```

Comment:
Am curious if this is still needed given `libnvimgcodec` is in the `conda/environments` files which are updated automatically